### PR TITLE
Wrapping

### PR DESCRIPTION
**Pull request by petersteneteg**
Added support for Wrapping and Interpolation in layer and volume

### CODE REVIEW

File: modules/base/src/io/datvolumewriter.cpp [hunk 1]
```diff
@@ -66,27 +74,43 @@ void DatVolumeWriter::writeData(const Volume* data, const std::string filePath)
...
+        [&](std::string_view key, InterpolationType val) { fmt::print(ss, "{}: {}", key, val); },
+        [&](std::string_view key, const SwizzleMask& mask) {
+            fmt::print(ss, "{}: {}{}{}{}", key, mask[0], mask[1], mask[2], mask[3]);
+        },
+        [&](std::string_view key, const Wrapping3D& wrapping) {
+            fmt::print(ss, "{}: {} {} {}", key, wrapping[0], wrapping[1], wrapping[2]);
+        },
+        [&](std::string_view key, const auto& vec) {
+            fmt::print(ss, "{}: {}", key, fmt::join(begin(vec), end(vec), " "));
+        }};
```
Comment (martinfalk) [inline modules/base/src/io/datvolumewriter.cpp:89]: that's pretty neat

**Review by martinfalk — APPROVED** [review]
looks good

**Comment by martinfalk** [comment]
Takes care of #771

File: modules/base/src/io/datvolumewriter.cpp (hunk 1 above)
Comment (petersteneteg) [reply modules/base/src/io/datvolumewriter.cpp:89]: yes

**Pull request merged**